Dockerfile

```diff
@@ -1,22 +1,44 @@
-FROM tomcat:8.5-jdk8-openjdk
-MAINTAINER geosolutions<info@geo-solutions.it>
+FROM tomcat:9-jdk11-openjdk AS mother
+LABEL maintainer="Alessandro Parma<alessandro.parma@geo-solutions.it>"
+ARG MAPSTORE_WEBAPP_SRC="./.placeholder"
+ADD "${MAPSTORE_WEBAPP_SRC}" "/mapstore/"
+
+COPY ./docker/* /mapstore/docker/
+WORKDIR /mapstore
+
+FROM tomcat:9-jdk11-openjdk
 
 # Tomcat specific options
 ENV CATALINA_BASE "$CATALINA_HOME"
-ENV JAVA_OPTS="${JAVA_OPTS}  -Xms512m -Xmx512m -XX:MaxPermSize=128m"
+ENV MAPSTORE_WEBAPP_DST="${CATALINA_BASE}/webapps"
+ENV INITIAL_MEMORY="512m"
+ENV MAXIMUM_MEMORY="512m"
+ENV JAVA_OPTS="${JAVA_OPTS}  -Xms${INITIAL_MEMORY} -Xmx${MAXIMUM_MEMORY} -XX:MaxPermSize=128m"
+ENV GEOSTORE_OVR_OPT="-Dgeostore-ovr=${CATALINA_BASE}/conf/${OVR} \
+                      -Duser.timezone=UTC"
```

Comment:
"-Duser.timezone=UTC" is not directly related with GEOSTORE

Reply:
resolved here https://github.com/geosolutions-it/MapStore2/pull/6490/commits/e597002a1188e6809cdeabaaca0129fb396e4121